docs/notebooks/Chemical_Structure_Alignment.ipynb

```diff
@@ -0,0 +1,277 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Chemical Structure Alignment"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Molecular alignment is a fundamental problem in cheminformatics and can be usedfor  structure  determination,  similarity  based  searching,  and  ligand-baseddrug design et al. This problem can be solved by orthogonal Procrustes when given two matrices representing three-dimensional coordinates.  The code block below showsthe ease-of-use of the `Procrustes` library for protein structure alignment, one of themost fundamental problems in structural biophysics."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here, we are going to use `procrustes.rotational` for protein structure alignment as an example, which is one of the fundamental problems in structural biology and structured-based drug design. A well-known structure alignment method is the *Kabsch* algorithm, which involves three steps: \n",
+    "1. translation of the structures to have a common centroid;\n",
+    "2. computation of the coordinate covariance matrix;\n",
+    "3. finding the optimal rotation matrix. \n",
+    "\n",
+    "This is simply a rotational Procrustes problem."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The human deoyhemoglobin X-ray crystal structure is used here (PDB ID: 2HHB), which allows comparison to the  BiomolecularStructures package (https://github.com/hng/BiomolecularStructures.jl). \n",
+    "The structure has cyclic-C2 global symmetry, where chain A (red)\n",
+    "and chain C (blue) are hemoglobin (deoxy) alpha chains, and chain B and D are hemoglobin deoxy beta chains (grey) (**Fig (i)**).  The rotational-orthogonal Procrustes will be used to align the Cα atoms in different chains of the protein."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "![Fig. 1. protein structure alignment with rotational Procrustes.](notebook_data/chemical_strcuture_alignment/protein_alignment.png \"Fig. 1. protein structure alignment with rotational Procrustes.\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# only run this cell if you need to install the dependices\n",
+    "\n",
+    "# install procrustes\n",
+    "!pip install git+https://github.com/theochem/procrustes.git@master\n",
+    "# iodata\n",
+    "!pip install git+https://github.com/theochem/iodata.git@master"
+   ]
+  },
```

Comment:
I think this is not needed. The user should install the package locally and the Binder would have it installed for them automatically too. If someone likes to run these examples through Colab, etc. I think it's their job to install it. What do you think @PaulWAyers?

Reply:
According to what I have learned, MyBinder can only install `Procrustes` automatically and those in `requirements.txt`, but not for other dependencies in the notebook. That's saying we will also need `IOData` and `matplotlib` if we want a user to run the notebooks successfully. I have experimented that adding an extra `requirements.txt` or `environment.yml` under `notebooks` folder won't resolve the problem because Binder follows a "1 repository -> 1 environment" rule. 

So, we have two options, 
1. Add `IOData` and `matplotlib` to `requirements.txt` which is not a wise choice. We will add redundant dependencies if someone wants to install our package.
2. Manually install them within the notebooks when needed.

For now, I am in favor of the second way. So, for this issue, what we can do is to only remove the line to install `Procrustes`, but have to keep the one for `IOData` and `matplotlib`.

For `IOData`, we have to use the source to install the package now as the one in PyPi does not include the feature to work with PDB files.

Any suggestions here? @FarnazH @PaulWAyers

Reply:
This is addressed in 0a25cb4.